index.html

```diff
@@ -0,0 +1,130 @@
+                <div class="card-display">
+                <div class="program-image">
+                    <img src="./images/exhibitionicon (2).svg" alt="lecture icon">
+                    <p class="program-name">Lecture</p>
+                    <p class="program-text">
+                        Listen to honourable speakers from various locations about the mission of hennahubuh mehndi creations 
+                    </p> 
+                </div>
+                </div>
+                <div class="card-display">
+                <div class="program-image">
+                    <img class="icon-image" src="./images/networkicon (3).svg" alt="exhibition icon">
+                    <p class="program-name">Exhibition</p>
+                    <p class="program-text">
+                        Showcasing models with various mehndi designs and apreciating other mehndi creativities on materials 
+                    </p> </div>
+                </div>
+                <div class="card-display">
+                <div class="program-image">
+                    <img class="icon-image" src="./images/networkicon (4).svg" alt="Workshop Icon">
+                    <p class="program-name">Workshop</p>
+                    <p class="program-text">
+                        Workshops on the various creativity done by HennaHubUk academy
+                    </p>
+                </div> 
+                </div>
+                <div class="card-display">
+                <div class="program-image">
+                    <img class="icon-image" src= "./images/lectureicon (1).svg" alt="Networking icon">
+                    <p class="program-name">Networking</p>
+                    <p class="program-text">
+                        Get connected with fan around the world and make new friends even after the summit
+                    </p>
+                </div> 
+                </div>  
+            </section>
+            <button> JOIN HENNAHUB SUMMIT 2023</button>
+        </div>
+        <a href="#">SEE THE WHOLE PROGRAM</a>
+    </div>
+    <section class="featured-guests" id="featured-guests">
+        <h2>Featured Speakers</h2>
+        <hr class="hr-line">
+        <div class="grid-cards"></div>
+        <div class="speaker-holder"></div>
+      </section>
+      <div class="more-info">
+        More <i class="fa-sharp fa-solid fa-chevron-down"></i>
+      </div>
+      <div class="less-info">
+        Less <i class="fa-sharp fa-solid fa-chevron-up"></i>
+      </div>
+      <div class="sponsors">
+        <p>Partner</p>
+        <hr>
+        <div class="spons">
+          <img src="./images/mozilla.PNG" alt="mozilla_logo">
+          <img src="./images/google.PNG" alt="google_logo">
+          <img src="./images/naver.PNG" alt="naver_logo">
+          <img src="./images/fancybraidslogo1.png" alt="fancy_logo">
+          <!-- <imc: src="./images/daumkakao.PNG" alt="daumkakao_logo">
+          <img src="./images/airbnb.PNG" alt="airbnb_logo"> -->
```

Comment:
- [ ] Please do not push commented-out code to your code base. It makes your project looks messy and unclear, therefore, I suggest you remove all commented-out code to make your project looks more professional.